lib/chef/resource/aws_rds_parameter_group.rb

```diff
@@ -24,7 +24,7 @@ def aws_object
       parameters += more_results[:parameters]
       marker = more_results[:marker]
     end
-    # object[:parameters] = parameters
+    driver.rds.reset_db_parameter_group(db_parameter_group_name: name, parameters: parameters)
```

Comment:
What is the reason for doing this in the `aws_object` method? I would think something like this would occur in the provider's update logic.

Reply:
@tyler-ball update limit is 20 that's why I am using `reset_db_parameter_group` here this is pretty faster to reset all parameters.